server/system/appCluster.js

```diff
@@ -66,4 +67,5 @@ module.exports = async () => {
   // ====== schedulers
   await RecordPreviewCleanup.init()
   await ExpiredJwtTokensCleanup.init()
+  await TempFilesCleanup.init()
 }
```

Comment:
Why don't we put the path to the tempFolder in .env and initialize goth the logger and the TempFilesCleanup with its value?

Reply:
very good idea, thanks!

Reply:
Cool. Please update the read me as well